Form/Handler/AuthorizeFormHandler.php

```diff
@@ -39,15 +38,15 @@ public function isRejected()
         return !$this->form->getData()->accepted;
     }
 
-    public function process()
+    public function process(Request $request)
```

Comment:
Changing this signature is a BC break. What about injecting the RequestStack instead, to be able to support newer Symfony versions without breaking BC ?